### PR TITLE
feat: added form page to Ecom

### PR DESCRIPTION
**Pull request by ooreally**
- added form to Ecom
- linked different pages
Here's the ss of page
image
image

### CODE REVIEW

**Review by vikrantkode — COMMENTED** [review]
looking good
_UI is good and clean

File: css/index.css [hunk 1]
```diff
@@ -7,6 +7,7 @@
     --smokewhite-color: whitesmoke;
     --border-color-white: #f3f4f6;
     --secondary-btn-color:#9333EA;
+    --light-gray-color: rgb(216, 216, 216);
```
Comment (sharmayatendra) [inline css/index.css:10]: try to maintain consistency either use hex or rgb

**Review by sharmayatendra — APPROVED** [review]
can make the suggested changes. else good to merge

**Comment by SuchitaGhadge** [comment]
UI looks good. 
On login card "account" spelling is incorrect. 
You can add a underline or some highlighted text color  to "Sign in here" and "create an account".  So it will look like a clickable links.

**Comment by SuchitaGhadge** [comment]
<details><summary>Comment has been resolved</summary>

Account spelling is wrong on login page. 
You might want to add some underline or test highlight on "create new account" and "sign in here" links to make it look clickable.
UI looks good.
<details><summary>Browser metadata</summary>





```
Path:      /pages/form/login.html
Browser:   Chrome 98.0.4758.102 on Windows 10
Viewport:  1536 x 746 @1.25x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-5--g-mart.netlify.app%2Fpages%2Fform%2Flogin.html&os=Windows&os_version=10&browser=Chrome&browser_version=98.0&resolution=1536x746&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-5--g-mart.netlify.app/pages/form/login.html) · [Mark as Unresolved](https://app.netlify.com/cdp/resolve?deployID=6212da0f295abc000824e9b6&commentID=6213117b335c6a5ee0518d19&resolution=unresolved)

</details>

File: css/index.css (hunk 1 above)
Comment (ooreally) [reply css/index.css:10]: ok

**Pull request merged**